### PR TITLE
Expose equivalentwords to license-list-data

### PR DESCRIPTION
**Pull request by m1kit**
This will close https://github.com/spdx/license-list-XML/issues/1225.

I renamed the file to `equivalentwords.csv` to make it clear the file is in a form of `csv`, but I can revert it if there is a problem.

### CODE REVIEW

**Comment by m1kit** [comment]
Oh, the author of commits seem to be wrong... I'm fixing

File: Makefile [hunk 1]
```diff
@@ -51,7 +56,7 @@ licenseListPublisher-%.jar.asc:
 
 .PHONY: licenseListPublisher-%.jar-valid
 licenseListPublisher-%.jar-valid: licenseListPublisher-%.jar.asc licenseListPublisher-%.jar goneall.gpg
-	gpg --verify --no-default-keyring --keyring ./goneall.gpg $<
```
Comment (m1kit) [inline Makefile:54]: I forgot to uncomment this... fixing.

**Review by goneall — APPROVED** [review]
Changes look good to me - I'm OK with changing the extension to csv - but we should check with other members of the legal team before merging.

@swinslow @jlovejoy  Let us know if you agree with this proposed change - both the renaming of the file and adding the file to the website for public access

**Comment by m1kit** [comment]
I noticed we need to update 8.1.1 in https://spdx.dev/license-list/matching-guidelines/ if we change the extension. I guess it's not  easy, so should I revert this chang,,,?

**Comment by goneall** [comment]
> I noticed we need to update 8.1.1 in https://spdx.dev/license-list/matching-guidelines/ if we change the extension. I guess it's not easy, so should I revert the change...?

Good catch - although it isn't too difficult to update 8.1.1, there may be other references and possibly even programmatic access to the current file location.  So, perhaps we should leave the extension at .txt.

**Comment by m1kit** [comment]
I reverted the change.

Although this is not mandatory, we should update 8.1.1 anyway to refer `http://spdx.org/licenses/equivalentwords.txt` since [license-list-XML](https://github.com/spdx/license-list-XML) is not public API. I'll make a PR in [spdx/spdx-spec](https://github.com/spdx/spdx-spec) for it, if needed.

**Comment by goneall** [comment]
Thanks @m1kit I'll go ahead and merge this.

@swinslow I don't think this will change any of the deployment operations, but just wanted you to be aware it will now be included on the website.

**Pull request merged**